simpler/connectors.py

```diff
@@ -303,6 +272,20 @@ def insert_all(self, table: str, rows: Union[List[dict], List[tuple]], tuple_row
 				','.join(values for _ in rows)
 			)
 			params = [insertion[key] for insertion in rows for key in keys]
+		if self.engine == 'postgre':

```

Comment:
This would disable the behaviour when tuple_rows is True, and would execute the previous if-else block and then dispose its value. I'd suggest either (a) merging the whole new block into the already existing if-else or (b) move this above so that you do

```py
if self.engine == 'postgre':
    if tuple_rows:
        # stuff
    else:
        # stuff
else:
    if tuple_rows:
        # stuff
    else:
        # stuff
```

The second option would create too much repetition, I'd sugest trying to merge both and then testing it both with tuple rows `True` and `False`.